### ISSUE TITLE
edit bot using valid pairs

### ISSUE THREAD

**Issue by marianolatorre** [3commas-io/3commas-official-api-docs#94]
Let's say I wanted to edit the list of pairs of a composite bot.

If I know the bot id, is there a way to tell which pairs are "valid" for that bot id? 

I have 3 things that come up:
1) if the bot already had BTC pairs, how can tell that before editing the bot to make sure that I only add BTC pairs?
2) given that the exchange can be changed after the bot has been already saved, how can I tell which pairs are available for that particular pair.
3) how can I tell that the bot is actually simple and therefore I won't be able to add multiple pairs.

In summary and as an example: if an existing bot is composite, using Binance.com and BTC pairs, I would like to have an array of all BTC pairs that can be added to that particular bot. And if the bot is simple then maybe an error or a flag somewhere to become aware.
